daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -140,10 +149,11 @@ instance IsParties Party where
 instance IsParties [Party] where
   toParties ps = ps
 
--- | HIDE
 class Template c => TemplateKey c k | c -> k where
+    -- | The key of a contract.
```

Comment:
This is a bit light. Could you add something saying what a key is? Doesn't need to be much - eg allowing unique lookup, can persist across changes of the contract (unlike the id)